README.md

```diff
@@ -5,25 +5,13 @@ Develop: [![Build Status](https://travis-ci.org/usdot-jpo-ode/jpo-ode.svg?branch
 # jpo-ode
 US Department of Transportation Joint Program office (JPO) Operational Data Environment (ODE)
 
-![Architecture Diagram](images/ODE_Architecture.png)
+![System Diagram](images/overview.png)
```

Comment:
I removed this diagram from our design doc because it is too SE MI focused. Let's not use it for now until we come up with a more accurate one for JPO ODE. We also need to put content around it. Having just a diagram doesn't look right.